card/spec/models/card_spec.rb

```diff
@@ -138,12 +138,12 @@ def assert_stable card1
   end
 
   it 'update_should_create_subcards_as_wagn_bot_if_missing_subcard_permissions' do
-    Card.create :name=>'peel'
+    Card.create name: 'peel'
     Card::Auth.current_id = Card::AnonymousID
     expect(Card['Banana']).not_to be
     expect(Card['Basic'].ok?(:create)).to be_falsey, "anon can't creat"
 
-    Card.create! :type=>"Fruit", :name=>'Banana', :subcards=>{ "+peel" => { :content => "yellow" }}
+    Card.create! type: "Fruit", name: 'Banana', subcards: { "+peel" => { content: "yellow" }}
```

Comment:
Line is too long. [93/80]<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.